### DISCUSSION TITLE
What is the difference between `50256-consistent-oneword` and `50256-consistent`?

### DISCUSSION THREAD

**Discussion by Calvinnncy97** [alasdairforsythe/tokenmonster#22] [Q&A]
Hey guys,

Specifically, I would like to ask which flags are set to train these 2 tokenizers? I can't find any flags that force tokenizer to have only 1 word tokens. 

Thank you.

**Comment by alasdairforsythe** [comment]
https://github.com/alasdairforsythe/tokenmonster/issues/24
